### ISSUE TITLE
Remove scrypt and bcrypt from npm dependencies

### ISSUE THREAD

**Issue by xafizoff** [macchiato-framework/macchiato-auth#2]
Hi.
I think dependency on scrypt and bcrypt is bad because:
* my investigation showed that `macchiato-auth` does not use them directly
* IMO password encryption is out of scope of this library
* this libs require python to be installed (main reason why I want to get rid of them)
Am I missing something?

Eventually, I've ported access rules from buddy-auth. I can PR my fork, if you accept this issue.

**Comment by yogthos** [comment]
That sounds reasonable to me, and a PR would be welcome.

**Comment by xafizoff** [comment]
Currently, using cljs v1.9.456, lein test fails with the following error
```
Library name must be specified as a symbol in :require / :require-macros; offending spec: ["fs" :as fs]
```
With the cljs v1.9.946 tests passed with no errors. Should we switch to another version of cljs? What do you think about this?

**Comment by yogthos** [comment]
Yeah, I think moving  up to the latest cljs would be best.

**Comment by xafizoff** [comment]
fixed in #3
